src/factory.js

```diff
@@ -1,51 +1,51 @@
-import React, { Component } from "react";
-import PropTypes from "prop-types";
-import isNumeric from "fast-isnumeric";
-import objectAssign from "object-assign";
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import isNumeric from 'fast-isnumeric';
+import objectAssign from 'object-assign';
 // import throttle from "throttle-debounce/throttle";
 
```

Comment:
can we delete this commented out import

Reply:
agreed though I'll let the future maintainer do it. Maybe they'll want to create a throttling issue or something.